### PR TITLE
[JN-1447] recurring survey admin view

### PR DESCRIPTION
**Pull request by devonbush**
#### DESCRIPTION (include screenshots, and mobile screenshots for participant UX)

Very basic admin view of surveys with multiple responses.  Also includes populate capability.  This will undoubtedly be refined/expanded in future work.

image

This does not yet include export of multiple responses, or any participant UX affordances

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

1. redeploy apiAdminApp
2. repopulate demo
3. go to `https://localhost:3000/demo/studies/heartdemo/env/sandbox/participants/HDSALK`
4. click on the "lifestyle" survey
5. confirm you see two responses
6. check that you can view/edit the multiple responses as you would expect

### CODE REVIEW

File: populate/src/main/resources/seed/portals/demo/studies/heartdemo/enrollees/jsalk.json [hunk 1]
```diff
@@ -48,6 +48,26 @@
       ],
       "currentPageNo": 1,
       "complete": true,
+      "submittedHoursAgo": 247
+    }, {
+      "surveyStableId": "hd_hd_lifestyle",
+      "surveyVersion": 1,
+      "answerPopDtos": [
+        { "questionStableId": "hd_hd_lifestyle_daysVigorousActivity", "stringValue": "2days"},
```
Comment (connorlbark) [inline populate/src/main/resources/seed/portals/demo/studies/heartdemo/enrollees/jsalk.json:56]: I think this should be `2Days`, right now it doesn't display properly in the preview

File: populate/src/main/resources/seed/portals/demo/studies/heartdemo/enrollees/jsalk.json [hunk 2]
```diff
@@ -48,6 +48,26 @@
...
+        { "questionStableId": "hd_hd_lifestyle_vigorousActivityMinutes", "stringValue": "30"}
+      ],
+      "currentPageNo": 1,
+      "complete": true,
+      "submittedHoursAgo": 247
+    }, {
+      "surveyStableId": "hd_hd_lifestyle",
+      "surveyVersion": 1,
+      "answerPopDtos": [
+        { "questionStableId": "hd_hd_lifestyle_daysVigorousActivity", "stringValue": "3days"},
```
Comment (connorlbark) [inline populate/src/main/resources/seed/portals/demo/studies/heartdemo/enrollees/jsalk.json:66]: Same here, `3Days`

**Review by connorlbark — COMMENTED** [review]
Everything looks great and I really like the UX - unobtrusive but clear. I just ran into a weird bug after editing the older response. It seemed, for that question, to only show the history of the older response despite being able to show both histories separately before I made the edit:

https://github.com/user-attachments/assets/ee760096-a8fd-4fa7-beed-21b1f4098f82

**Comment by devonbush** [comment]
great find on that bug!  I think we are grabbing all answers, and then just sorting by survey/questionStableId, not by responseId.  Will fix

**Review by MatthewBemis — APPROVED** [review]
Looks great, pre-approving this pending bug fix

**Review by connorlbark — APPROVED** [review]
Bug is fixed!

**Comment by devonbush** [comment]
Sonar spots are just for using pseudorandom numbers for synthetic participants.  This is not a security risk

**Pull request merged**